src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -191,7 +194,7 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             </div>
           </div>
         </div>
-        <div id="wi-detail-desc" class="col-md-12 description" *ngIf="!loggedIn">
+        <div id="wi-detail-desc" class="col-md-12 description" *ngIf="!loggedIn || addNewWI">
```

Comment:
Same logic as I explained up there. I don't think addNewWI flag is needed anywhere.